modules/pricer/src/main/java/com/opengamma/strata/pricer/impl/volatility/local/LocalVolatilityCalculator.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (C) 2016 - present by OpenGamma Inc. and the OpenGamma group of companies
+ * 
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.pricer.impl.volatility.local;
+
+import java.util.function.Function;
+
+import com.opengamma.strata.market.surface.Surface;
+
+/**
+ * Local volatility calculation.
+ */
+public interface LocalVolatilityCalculator {
+
+  /**
+   * Computes local volatility surface from call price surface. 
+   * 
+   * @param callPrcieSurface  the price surface
+   * @param spot  the spot
+   * @param interestRate  the interest rate
```

Comment:
Add precision on IR: is it period rates or instanteneous forward, zero-coupon? which convention?